diff --git a/node_modules/react-native-vector-icons/lib/tab-bar-item-ios.js b/node_modules/react-native-vector-icons/lib/tab-bar-item-ios.js
index fb53b6d..44b24f2 100644
--- a/node_modules/react-native-vector-icons/lib/tab-bar-item-ios.js
+++ b/node_modules/react-native-vector-icons/lib/tab-bar-item-ios.js
@@ -87,7 +87,7 @@ export default function createTabBarItemIOSComponent(
     }
 
     render() {
-      return <TabBarIOS.Item {...this.state} {...this.props} />;
+      return <></>;
     }
   };
 }
