diff --git a/node_modules/react-native-video/Video.js b/node_modules/react-native-video/Video.js
index 6b84021..c863c3a 100644
--- a/node_modules/react-native-video/Video.js
+++ b/node_modules/react-native-video/Video.js
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { StyleSheet, requireNativeComponent, NativeModules, View, ViewPropTypes, Image, Platform, findNodeHandle } from 'react-native';
-import resolveAssetSource from 'react-native/Libraries/Image/resolveAssetSource';
+//import resolveAssetSource from 'react-native/Libraries/Image/resolveAssetSource';
 import TextTrackType from './TextTrackType';
 import FilterType from './FilterType';
 import DRMType from './DRMType';
@@ -262,7 +262,7 @@ export default class Video extends Component {
 
   render() {
     const resizeMode = this.props.resizeMode;
-    const source = resolveAssetSource(this.props.source) || {};
+    const source = Image.resolveAssetSource(this.props.source) || {};
     const shouldCache = !source.__packager_asset;
 
     let uri = source.uri || '';
